server/build-plugins.js

```diff
@@ -42,15 +85,30 @@ const prepareModules = async (plugins = [], local = true) => {
   let exportsText = 'export default {';
   let installPackages = [];
 
+  // Create the index.js files for exposing the plugins
   plugins.forEach(name => {
     const packageName = getPackageName(name);
-    const camelized = camelize(packageName);
-    const modulePath = local ? `./${packageName}` : packageName;
-    if (!local) installPackages.push(name);
-    importsText += `import * as ${camelized} from '${modulePath}';\n`;
-    exportsText += `${camelized},`;
+    try {
+      const camelized = camelize(packageName);
+      let modulePath;
+      if (fs.existsSync(resolve(pluginsPath, 'local', packageName)) && local)
+        // maintain support for plugins in plugins/local dir
+        modulePath = `./${packageName}`;
+      // set import to alias for bpanel's local_plugins dir set in webpack
+      else modulePath = local ? `&local/${packageName}` : packageName;
```

Comment:
the `&local` prefix is recognized by webpack as an alias to indicate a plugin in `[BPANEL_PREFIX]/local_plugins`

Reply:
We're maintaining support for the `webapp/plugins/local` directory as well by using the old convention for the import path if that plugin directory exists.

Reply:
The comments are more or less saying that. Just wanted to point it out explicitly for review. Will clean up the wording a bit to make it more clear.